src/extension/ScriptRunner.ts

```diff
@@ -257,9 +258,22 @@ export class ScriptRunner {
         this.removeRunningScript(pid)
         // when there is a parse error, post it to PROBLEMS and send 'parse-error' to results
         PostProblems.postProblems(confFile)
+
+        const curLog = await this.getLogFilePath(confFile, ts)
+        let curLogStr = ''
+        if (curLog) {
+          try {
+            // check if file curLog exists
+            await workspace.fs.stat(curLog)
+            curLogStr = curLog.path
+          } catch (e) {}
```

Comment:
these empty catch blocks are really strange to me and i don't like them in general.

Reply:
if the stat function fails that means that the file doesn't exists so I send empty string instead of path... it's not a mistake that's why it's empty

Reply:
if it fails you do nothing, cause the catch block is empty.
how do you know it failed so you could send an empty string?

Reply:
the default value before the try-catch is empty string, and if the stat function fails we get out of the "try" part before the `curLogStr = curLog.path` part so it stays empty string